src/common/translation/i18n/translations/fi.json

```diff
@@ -217,17 +214,49 @@
     "close": "Sulje selekoitu filtteri"
   },
   "boater_services": {
-    "guest_harbors": "Etsi satamia",
+    "guest_harbors": "Kaikki vierasvenesatamat",
+    "service_stations": "Veneiden huoltoasemat ja venekorjaamot",
+    "boat_launch": "Veneiden vesillelaskupaikat"
+  },
+  "seaside_experiences": {
+    "sightseeing": "Nähtävyydet",
     "restaurants": "Kaikki Helsingin merelliset ravintolat",
-    "rental_boats": "Vuokraa vene käyttöösi"
+    "small_boats": "Soutuveneily, suppilautailu, kajakit ja kanootit",
+    "city_boats": "Kaupunkiveneily",
+    "boat_rental": "Vuokraveneet"
   },
   "footer": {
     "boating": "Kaupungin veneilysivut",
     "boating_url": "https://www.hel.fi/helsinki/fi/kulttuuri-ja-vapaa-aika/ulkoilu/veneily",
     "sea_rescue": "Meripelastusohjeet",
-    "sea_rescue_url": "https://www.hel.fi/helsinki/fi/kulttuuri-ja-vapaa-aika/ulkoilu/veneily/lisatietoa-veneilysta/meripelastusohjeet/"
+    "sea_rescue_url": "https://www.hel.fi/helsinki/fi/kulttuuri-ja-vapaa-aika/ulkoilu/veneily/lisatietoa-veneilysta/meripelastusohjeet/",
+    "feedback": "Anna palautetta",
+    "feedback_url": "https://www.hel.fi/helsinki/fi/kaupunki-ja-hallinto/osallistu-ja-vaikuta/palaute/anna-palautetta"
   },
   "language_select": {
     "select_language": "Valitse kieli"
+  },
+  "suggestion_form": {
+    "heading": "Ehdota lisättävä palvelu Ahtiin",
+    "name": "Palvelun nimi",
+    "description": "Palvelun kuvaus",
+    "short_description": "Palvelun lyhyt kuvaus",
+    "website": "Palvelun verkkosivu",
+    "address": "Palvelun osoite",
+    "phone": "Palvelun puhelinnumero",
+    "email": "Palvelun sähköpostiosoite",
+    "category": "Palvelun kategoria",
+    "tags": "Lisää palvelulle tageja alla olevasta listasta",
+    "submit": "Lähetä palvelun tiedot",
+    "coordinates": "Valitse kartalta palvelun sijainti",
+    "selected_tags": "Valitut tagit",
+    "no_selected_tags": "Ei yhtäkään valittua tagia",
+    "on_submit": "Ehdotuksesi on vastaanotettu!",
+    "errors": {
+      "required": "Tämä kenttä on pakollinen",
+      "email": "Kentän arvo ei ole kelvollinen sähköposti",
+      "translated_field_required": "Suomalainen käännös tälle kentälle on pakollinen",
+      "check_form": "Tietoja ei voitu lähettä. Tarkista syöttämäsi tiedot."
+    }
```

Comment:
Swedish translations don't seem to be added.

Reply:
Swedish translations are still WIP. Will probably not make it into this PR.